package.json

```
@@ -190,7 +190,7 @@
     }
   },
   "engines": {
-    "node": ">= 18"
+    "node": ">= 20.19"
```

@MichalBryxi why not 
```suggestion
    "node": ">= 20.x"
```

---

Anything below `20.19` results in following error in consuming app for me:

```
> docs-app-for-ember-intl@1.5.6 start /Users/michal/ember/ember-intl/docs/ember-intl
> ember serve

require() of ES Module /Users/michal/ember/ember-intl/node_modules/.pnpm/@handlebars+parser@file+..+handlebars-parser/node_modules/@handlebars/parser/dist/cjs/index.js not supported.
Instead change the require of index.js in null to a dynamic import() which is available in all CommonJS modules.


Stack Trace and Error Report: /var/folders/8q/yrjx_8s115q43r8w5qd90_m00000gn/T/error.dump.09dfcadbf87770bc232cc47d14bf82ae.log
 ELIFECYCLE  Command failed with exit code 1.
node --version
v20.18.0
```

IDK how to signify that the _consuming_ app should have certain node version, but sounds reasonable to impose minimal node version _also_ on this repo to not bump into the same set of issues in, say, the testing app (which I _think_ is actually happening).

Related [Discord convo](https://discord.com/channels/480462759797063690/1361741610735370240/1361791032714465482).

---

Ah yeah, this addon's CI itself is having issue with `node < 20.19`: https://github.com/ember-learn/ember-cli-addon-docs/actions/runs/14612606117/job/40993520690?pr=1672